src/progressbar/progressbar.ts

```diff
@@ -38,10 +37,12 @@ export class NgbProgressbar {
 
   @Input()
   set value(value: number) {
```

Comment:
"Yes". You could parseInt on max to convert it to int. With the current implementation, a `50 / "100"` works, but perhaps not elegant.